Episerver/5.6.0/layerreferences/ref_metadefinitions.dcl

```diff
@@ -0,0 +1,142 @@
+data meta_group_folder asset_50000 {
+	name = 'Asset'
+	parent_id = 0
+}
+
+data meta_group_folder shared_50004 {
+	name = 'Shared'
+	parent_id = data.meta_group_folder.asset_50000.id
+}
+
+data meta_group_folder video_50006 {
+	name = 'Video'
+	parent_id = data.meta_group_folder.asset_50000.id
+}
+
+data meta_group_folder users_and_usergroups_50001 {
+    parent_id = 0
+    name = 'Users and usergroups'
+}
+
+data meta_group_folder users_and_usergroups_shared {
+    parent_id = data.meta_group_folder.users_and_usergroups_50001.id
+    name = 'Shared'
+}
+
+data metafield_group asset_info_10025 {
+	name = 'Asset info'
+	folder_id = data.meta_group_folder.shared_50004.id
+    parent_group_id = 0
+}
+
+data metafield_group video_10027 {
+	name = 'Video'
+	folder_id = data.meta_group_folder.video_50006.id
+    parent_group_id = data.metafield_group.asset_info_10025.metafield_group_id
+    iterative = false
+    restrict_to_asset = 'Video'
+}
+
+data metafield_group_label video_50034 {
+    metafield_group_id = data.metafield_group.video_10027.metafield_group_id
+    label = 'Video'
+    language_id = 3
+}
+
+data metafield_group frontendgroup_10044 {
+	name = 'FrontendGroup'
+	folder_id = data.meta_group_folder.users_and_usergroups_shared.id
+	restrict_to_asset = 'All'
+}
+
+data metafield_group valid_download_qualities_10051 {
+	name = 'Valid Download Qualities'
+	folder_id = data.meta_group_folder.users_and_usergroups_shared.id
+	parent_group_id = data.metafield_group.frontendgroup_10044.metafield_group_id
+	iterative = true
+	restrict_to_asset = 'All'
+}
+data metafield_group user_config_50003 {
+	name = 'User Config'
+	folder_id = data.meta_group_folder.users_and_usergroups_shared.id
+	restrict_to_asset = 'All'
+}
+
+
+data combovalue_metafield format_10032 {
+	item_guid = 'f2a9baa4-655d-4cb3-8649-1692f56950d9'
+	name = 'Format'
+	group_id = data.metafield_group.video_10027.metafield_group_id
+}
+
+data combovalue_metafield rotation_50041 {
+	item_guid = 'c134e2d2-8caf-4739-bca0-767d34c7ac03'
+	name = 'Rotation'
+	group_id = data.metafield_group.video_10027.metafield_group_id
+}
+
+data string_metafield title_50181 {
+	item_guid = '5eb3eefc-a043-410f-89b0-29ed3ef37078'
```

Comment:
You have already specified item_guid on these, no need to specify more properties, since item_guid really should be unique.

Reply:
> You have already specified item_guid on these, no need to specify more properties, since item_guid really should be unique.

done